appcelerator-developer-relations/KitchenSink
# TIMOB-9948: BlackBerry: Make KitchenSink build for BB

Changelog:
- Recreate KitchenSink with Titanium Studio.
- Modify tiapp.xml to support blackberry.
- Added blackberry resources.

Test cases:
- Import KitchenSink into Tittanium Studio.
- Make sure it successfully builds and launches for Blackerry.
- Make sure it successfully builds and launches for Android, iOS.
